app/static/js/controllers/clusterController.js

```diff
@@ -200,6 +200,11 @@ angular.module('uluwatuControllers').controller('clusterController', ['$scope',
                 result.blueprintId = parseFloat(result.blueprintId);
 
                 var existingCluster = $filter('filter')($rootScope.clusters, {id: result.id}, true)[0];
+                if (result !== undefined && result.id !== undefined) {
```

Comment:
The backend may change the platformVariant (fall back to default version) and the create request only returns the StackId.